src/main/java/strategy/OperationHandler.java

```diff
@@ -0,0 +1,13 @@
+package strategy;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+
+public interface OperationHandler {
+    Map<String, Integer> currentBalanceByFruit = new HashMap<>();
+
+    Map<String, Integer> getCurrentBalanceByFruit(List<FruitTransaction> fruitTransactions,
+                                                  int index);
```

Comment:
It is not necessary to create additional Map
```suggestion
    void handle(FruitTransaction fruitTransaction);
```

Reply:
thanks for comment. I decided to keep method handle with return type Map<> (as it might be more useful) and work with list of FriutTransactions, not  with single FruitTransaction object, because it seems to be more effective on big data

Reply:
done